tests/ui/unnecessary_unwrap.stderr

```diff
@@ -0,0 +1,36 @@
+error: you checked before that `unwrap()` cannot fail, instead of checking and unwrapping, it's better to use `if let` or `match`
```

Comment:
Tests should be added to existing tests at `simple_conditionals.rs` or `complex_conditionals.rs`.

Reply:
>  to existing tests at simple_conditionals.rs or complex_conditionals.rs

As a first-time contributor, it was not easy to find these tests (and you can see that I _didn't_).

Reply:
I understand. Feel free to add a comment pointing to the tests. Personally I look for tests by searching all files for "you checked before that", which is definitely weird.